lib/libzpool/uzfs_io.c

```diff
@@ -310,3 +308,52 @@ uzfs_zvol_get_rebuild_status(zvol_state_t *zv)
 {
 	return (zv->zv_rebuild_status);
 }
+
+/*
+ * This assumes snapshot, so, no need of range lock
```

Comment:
yes. Actually, range lock for metadata is not needed since all the operation on metadata will go through ```uzfs_read_data``` and ```uzfs_write_data```.
In uzfs_get_io_diff, we are calling uzfs_read_metadata but that is on snapshot so no need of range lock.
I will remove range lock for metadata.